tests/cpp/test_sdpa_node.cpp

```diff
@@ -216,73 +319,6 @@ TEST_F(SDPATest, CausalAttn) {
   validateSdpaFwdOutputs(nvf_out, aten_out);
 }
 
-TEST_F(SDPATest, PairwiseLogicalDomainMap) {
```

Comment:
Removing this test, since it is redundant with the IDModel mapping checks included with all unit tests.

Reply:
I'd leave these since they are not exactly the same. There are some overlap, but seems like a negligible cost given that these properties are really important to make them right.